playbook/app/pb_kits/playbook/pb_date_picker/_date_picker.jsx

```diff
@@ -74,7 +74,7 @@ const DatePicker = (props: DatePickerProps) => {
     onChange = () => {},
     pickerId,
     placeholder = 'Select Date',
-    plugins = false,
+    plugins = "",
```

Comment:
```suggestion
    plugins = false,
```

Reply:
Keeping this false would be best as we will still check the value of this prop before enabling any plugins.